src/suites.rs

```diff
@@ -78,17 +78,17 @@ fn run_attempt(
     cancellation_token: &CancellationToken,
     output_directory: &Utf8Path,
 ) -> Result<(AttemptOutcome, Option<Utf8PathBuf>), Cancelled> {
-    let log_message_start = format!("Suite {}, attempt {}", id, attempt.index);
+    let log_message_start = format!("Plan {}, attempt {}", id, attempt.index);
 
     let run_outcome = match session
         .run(&RunSpec {
-            id: &format!("robotmk_suite_{}_attempt_{}", id, attempt.index),
+            id: &format!("robotmk_plan_{}_attempt_{}", id, attempt.index),
```

Comment:
What does this ID do again?

Reply:
This ID is used in the Windows task scheduler as the task ID